src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -58,50 +63,29 @@ public DefaultUserRepository(final ApplicationParams applicationParams,
     }
 
     @Override
-    public UserProfile getUserSettings() {
```

Comment:
getUserSettings has been renamed to getUserProfile.
Also, this is a method with complex orchestration logic more suitable for a Service than a Repository. 
I've introduced a UserService and move it there